src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx

```diff
@@ -82,14 +77,22 @@ With the Kubernetes integration you can also:
 
 These features are in addition to the data New Relic already reports for [containerized processes running on instrumented hosts](/docs/infrastructure/new-relic-infrastructure/data-instrumentation/docker-instrumentation-infrastructure).
 
-## Bring your cluster logs to New Relic [#k8s-logs]
-
-Our Kubernetes plugin for log monitoring can collect all your cluster's logs and send them to our platform, so that you can set up new alerts and charts. To set it up, check the <DoNotTranslate>**Gather Log Data**</DoNotTranslate> box in step 3 of [our install wizard](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#install-wizard). For detailed information, see our [log forwarding documentation](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) for Kubernetes.
-
 ## Check the source code [#source-code]
 
-This integration is open source software. That means you can [browse its source code](https://github.com/newrelic/nri-kubernetes) and send improvements, or you can create your own fork and build it. For more information, see the [README](https://github.com/newrelic/nri-kubernetes/blob/main/README.md).
+This integration is open source software. That means you can [browse its source code](https://github.com/newrelic/nri-kubernetes) and send in your own improvements, or you can create your own fork and build it. For more information, check out the [README](https://github.com/newrelic/nri-kubernetes/blob/main/README.md) file.
+
 
 <Callout variant="tip">
-Do you use Datadog to monitor Kubernetes but want to try out New Relic's monitoring capabilities for free? See our [guide on how to migrate from Datadog](/docs/journey-migration/migrating-from-dd/) to learn how.
-</Callout>
\ No newline at end of file
+Are you using Datadog to monitor Kubernetes but want to try out New Relic's monitoring capabilities for free? See our guide on [how to migrate from Datadog](/docs/journey-migration/migrating-from-dd/) to learn more.
+</Callout>
+
+
+## Choose your next step [#next-step]
```

Comment:
as next steps I would recommend adding the "Installation of the integration" as the first one